controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -852,6 +875,43 @@ export class OrganizationRepository {
     return result[0];
   }
 
+  public async queueOrganizationDeletion(input: {
+    organizationId: string;
+    queuedBy?: string;
+    deleteOrganizationQueue: DeleteOrganizationQueue;
+  }) {
+    const now = new Date();
```

Comment:
We should make this whole thing a migration so in case of the job cant be scheduled that the tx is rollbacked.